Gemfile.lock

```diff
@@ -9,7 +9,7 @@ GEM
   remote: https://rubygems.org/
   specs:
     builder (3.2.4)
-    rack (2.2.4)
+    rack (3.0.7)
```

Comment:
```suggestion
    rack (2.2.7)
```
Is what [our application](https://github.com/ualbertalib/jupiter/blob/0201c7eadfa3252baecbdffa61c0eed11df164ca/Gemfile.lock#L398) consuming this Gem is using for rack.  Seems like a safe compromise.